interface/vmcs_host/khronos/IL/OMX_Audio.h

```diff
@@ -1357,16 +1357,16 @@ typedef enum OMX_AUDIO_ECHOCANTYPE {
 } OMX_AUDIO_ECHOCANTYPE;
 
 
-/** Enable / Disable for echo cancelation, which removes undesired echo's
+/** Enable / Disable for echo cancellation, which removes undesired echo's
  *  from the audio
  * @ingroup effects
  */ 
-typedef struct OMX_AUDIO_CONFIG_ECHOCANCELATIONTYPE {
+typedef struct OMX_AUDIO_CONFIG_ECHOCANCELLATIONTYPE {
```

Comment:
Can you drop the cancellation fixes. This is from upstream Khronos header files and would break any code that used them. See https://www.khronos.org/registry/omxil/
I think leaving the khronos/IL header files as they are is best.